examples/llm_compression/torch/qat_with_nls_downstream/main.py

```diff
@@ -693,18 +781,27 @@ def form_batch(inputs: list[Tensor]):
             save_checkpoint(model, ckpt_file)
 
     # Start evaluation
+    finetuning_results = overall_result.get(
+        "finetuning_results",
+        {
+            "method": None,  # LoRA or NLS
+            "torch_result": None,
+            "ov_result": None,
+        },
+    )
+
+    # Torch model evaluation
```

Comment:
What about moving all evaluations after training to a separate function?

Reply:
@ljaljushkin Moved to a separate function to evaluate after training. Thanks!